kokioba95/clouvo_app
# #14-1 admin-function LAST-PR-#9

## Description / Purpose (Checkpoint)
This PR confirmation is the administrator function (& Associated page layout).

I fixed the problem.
< https://docs.google.com/spreadsheets/d/1Ear__g58irVlaI_tpzWUkf__JRcBz8u2d2DdRYKMiDk/edit?usp=sharing > <Backlog # 11>

# # To test this PR
【Admin】
1. Go to the login page (example: http: // localhost: 8086 / login).
2. Input user credentials (Admin)
3. Click Login
4. my_profile page
5. Go to the top page
6. Go to the event list page
7. Go to event info page
8. Press the Delete button.
9. (Go to Event list)
【User】
1. Go to the login page (example: http: // localhost: 8086 / login).
2. Input user credentials (User)
3. Click Login
4. my_profile page
5. Go to the top page
6. Go to the event list page
7. Go to event info page
8. Confirm that there is no Delete button.

**command**
# Php artisan migrate: reset

**screenshot**
 -< https://drive.google.com/drive/folders/1UX_at7VJu0OeH5Mb7Eo7ULK3hSw7Ws-W?usp=sharing >
